### DISCUSSION TITLE
How to remove the "home" tab when opening a pfd document in Sumatra?

### DISCUSSION THREAD

**Discussion by emi27777** [sumatrapdfreader/sumatrapdf#4270] [General]
How to remove the "home" tab when opening a pfd document in Sumatra?
![Captura de pantalla 2024-05-24 155753](https://github.com/sumatrapdfreader/sumatrapdf/assets/170760234/af5ab9f1-1a4d-4f6f-b509-01d2b186d73c)

**Comment by kjk** [comment]
Set `NoHomeTab = true` advanced setting. See https://www.sumatrapdfreader.org/settings/settings#NoHomeTab

**Reply by emi27777** [reply]
Solved, thank you very much!

**Reply by Zbeubertoundra** [reply]
Solved for me too ! Many thanks :) !
